gateway/src/apicast/policy/rate_limit/apicast-policy.json

```diff
@@ -64,7 +64,7 @@
           },
           "op": {
             "type": "string",
-            "enum": ["==", "!="]
+            "enum": ["==", "!=", "matches"]
```

Comment:
We should describe what is the format of "matches".

Reply:
Added :+1: